lte/gateway/c/core/oai/tasks/nas/ies/EpsQualityOfService.c

```diff
@@ -173,6 +192,19 @@ status_code_e eps_qos_bit_rate_ext_value(uint8_t br) {
   }
 }
 
+//------------------------------------------------------------------------------
+status_code_e eps_qos_bit_rate_ext2_value(uint8_t br) {
```

Comment:
add unit test

Reply:
I added this code because there exists similar functions for other octets but these functions are not being used anywhere. Can you please confirm if we are using "eps_qos_bit_rate_value", "eps_qos_bit_rate_ext_value" anywhere ?

Reply:
looks like we are not using them anywhere.

Reply:
Removed  before-mentioned functions.